src/main/java/org/sunbird/workflow/service/impl/OrganisationWorkFlowServiceImpl.java

```diff
@@ -38,4 +87,65 @@ public Response orgSearch(String rootOrg, String org, SearchCriteria criteria) {
         Response response = workflowService.applicationsSearch(rootOrg, org, criteria, Constants.ORG_SEARCH_ENABLED);
         return response;
     }
+
+    /**
+     * This method is responsible to check if the data with respect to the users already exists or not.
+     *
+     * @param key   - key is the parameter which is received in the wfRequest.
+     * @param value - value corresponding to the parameter received in the wfRequest.
+     * @return - Returns a boolean value true if the data already exists else false.
+     */
+    public boolean isUserDetailExists(String key, String value) {
+        Map<String, Object> reqMap = new HashMap<>();
+        reqMap.put(Constants.FILTERS, Collections.singletonMap(key, value));
+        Map<String, Object> requestObj = new HashMap<>();
+        requestObj.put(Constants.REQUEST, reqMap);
+        HashMap<String, String> headersValue = new HashMap<>();
+        headersValue.put(Constants.CONTENT_TYPE, Constants.APPLICATION_JSON);
+        headersValue.put(Constants.AUTHORIZATION, configuration.getSbApiKey());
+        try {
+            StringBuilder builder = new StringBuilder(configuration.getLmsServiceHost());
+            builder.append(configuration.getLmsUserSearchEndPoint());
+            Map<String, Object> response = (Map<String, Object>) requestServiceImpl
+                    .fetchResultUsingPost(builder, requestObj, Map.class, headersValue);
+            if (response != null && HttpStatus.OK.equals(response.get(Constants.RESPONSE_CODE))) {
+                Map<String, Object> map = (Map<String, Object>) response.get(Constants.RESULT);
+                if (map.get(Constants.RESPONSE) != null) {
+                    Map<String, Object> responseObj = (Map<String, Object>) map.get(Constants.RESPONSE);
+                    int count = (int) responseObj.get(Constants.COUNT);
+                    return count != 0;
+                }
+            }
+        } catch (Exception e) {
+            logger.info("There is a error occured while searching for the user details : " + e);
+        }
+        return true;
+    }
+
+    public boolean isOrgDetailExists(String key, String value) {
+        Map<String, Object> reqMap = new HashMap<>();
+        reqMap.put(Constants.FILTERS, Collections.singletonMap(key, value));
+        Map<String, Object> requestObj = new HashMap<>();
+        requestObj.put(Constants.REQUEST, reqMap);
+        HashMap<String, String> headersValue = new HashMap<>();
+        headersValue.put(Constants.CONTENT_TYPE, Constants.APPLICATION_JSON);
+        headersValue.put(Constants.AUTHORIZATION, configuration.getSbApiKey());
```

Comment:
check authorization is required

Reply:
Without Authorization the API is fetching the results so removed the Authorization from the header.